CHANGELOG.md

```diff
@@ -14,18 +14,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+* Service: Initial changes to port the service from Python to C#. [#1734](https://github.com/microsoft/onefuzz/pull/1734), [#1733](https://github.com/microsoft/onefuzz/pull/1733), [#1736](https://github.com/microsoft/onefuzz/pull/1736), [#1737](https://github.com/microsoft/onefuzz/pull/1737), [#1738](https://github.com/microsoft/onefuzz/pull/1738), [#1742](https://github.com/microsoft/onefuzz/pull/1742), [#1744](https://github.com/microsoft/onefuzz/pull/1744), [#1749](https://github.com/microsoft/onefuzz/pull/1749), [#1750](https://github.com/microsoft/onefuzz/pull/1750), [#1753](https://github.com/microsoft/onefuzz/pull/1753), [#1755](https://github.com/microsoft/onefuzz/pull/1755), [#1760](https://github.com/microsoft/onefuzz/pull/1760), [#1761](https://github.com/microsoft/onefuzz/pull/1761), [#1762](https://github.com/microsoft/onefuzz/pull/1762), [#1765](https://github.com/microsoft/onefuzz/pull/1765), [#1757](https://github.com/microsoft/onefuzz/pull/1757), [#1780](https://github.com/microsoft/onefuzz/pull/1780), [#1782](https://github.com/microsoft/onefuzz/pull/1782), [#1783](https://github.com/microsoft/onefuzz/pull/1783), [#1777](https://github.com/microsoft/onefuzz/pull/1777), [#1791](https://github.com/microsoft/onefuzz/pull/1791), [#1801](https://github.com/microsoft/onefuzz/pull/1801), [#1805](https://github.com/microsoft/onefuzz/pull/1805), [#1804](https://github.com/microsoft/onefuzz/pull/1804), [#1803](https://github.com/microsoft/onefuzz/pull/1803)
 * Service: Make sure the scale set nodes are unable to accept work while in the `setup` state. [#1731](https://github.com/microsoft/onefuzz/pull/1731)
 
 ### Changed
 * Agent: Reduce the logging level down from `warn` to `debug` when we are unable to parse an ASan log. [#1705](https://github.com/microsoft/onefuzz/pull/1705)
 * Service: Move the creation of the event grid topic to the deployment template from the `deploy.py` script. [#1591](https://github.com/microsoft/onefuzz/pull/1591)
 * Agent/Supervisor/Proxy: Updated multiple third-party Rust dependencies. [#1548](https://github.com/microsoft/onefuzz/pull/1548), [#1617](https://github.com/microsoft/onefuzz/pull/1617), [#1618](https://github.com/microsoft/onefuzz/pull/1618)
 * Service: Consolidate the two log analytics down to one. [#1679](https://github.com/microsoft/onefuzz/pull/1679)
+* Service: Updated resource name in bicep file to prevent name clash when deploying 5.3.0. [#1808](https://github.com/microsoft/onefuzz/pull/1808)
```

Comment:
```suggestion
* Service: Updated resource name in Bicep file to prevent name clash when deploying 5.3.0. [#1808](https://github.com/microsoft/onefuzz/pull/1808)
```